### PR TITLE
Crypto Today Web Application

### PR DESCRIPTION
**Pull request by Kamba56**
- Created react app and pushed it to GitHub.
- Loaded data from CoinCap API so that when the homepage loads, it displays various crypto.
- Added a search implementation to search using either name or symbol of cryptocurrency.
- Added a details page to show more details on click of either coin.
- Added test files to test the reducers and components.
- Recorded a loom video to show the features of the app. [Loom video](https://www.loom.com/share/0fc31a50fab5442bb12a9eef7115f4b9)

### CODE REVIEW

**Comment by Kamba56** [comment]
Hello, Please before you review, Note that not all changes are shown on this pull request because I created my repo with the develop branch and forgot to create a main branch from the on set.

File: src/__tests__/crypto.test.js [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+    },
+  ];
+
+  test('Test if coins are loaded correctly', () => {
+    expect(coinsReducer(initial, loadCoin(loadedCoin)).crypto).toEqual(loadedCoin);
+  });
+
+  test('Test if coins are loaded correctly', () => {
+    expect(coinsReducer(initial, searchCoin(searchedCoin)).search).toEqual(searchedCoin);
+  });
```
Comment (julie-ify) [inline src/__tests__/crypto.test.js]: - [x] One of your tests is failing. Kindly fix it.

  coin-test

**Review by julie-ify — CHANGES_REQUESTED** [review]
## Hi @Kamba56,

Good job so far!

## Highlights!
- Good feature branch name ✔️ 
- No linter errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 
- Good code organization and working app ✔️
- Descriptive README ✔️ 

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

# Status: Required Changes ♻️⛔️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@julie-ify` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/__tests__/crypto.test.js (hunk 1 above)
Comment (Kamba56) [reply src/__tests__/crypto.test.js]: Thank you for your review

**Review by julie-ify — APPROVED** [review]
Hi @Kamba56,

## Highlights!
- Good feature branch name ✔️ 
- No linter errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 
- Good code organization and working app ✔️
- Descriptive README ✔️ 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Status: Approved ✔️ 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**